pkg/datapath/loader/netlink.go

```diff
@@ -745,6 +745,9 @@ func ensureDevice(sysctl sysctl.Sysctl, attrs netlink.Link) (netlink.Link, error
 	l, err := netlink.LinkByName(name)
 	if err != nil {
 		if err := netlink.LinkAdd(attrs); err != nil {
+			if errors.Is(err, unix.ENOTSUP) {
+				err = fmt.Errorf("%w, maybe module for %s is not loaded?", err, attrs.Type())
```

Comment:
```suggestion
				err = fmt.Errorf("%w, maybe kernel module for %s is not available?", err, attrs.Type())
```

I *think* the kernel should be smart enough to auto-load the module when needed.

Reply:
yap, makes sense. Updated.